CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
```

Comment:
Can you explain why you needed to link `openssl`? I haven't needed to do this, and it's generally frowned upon.